contracts/src/utils/proxy/ProxyFactory.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity ^0.8.8;
+
+import {ProxyLib} from "./ProxyLib.sol";
+
+/// @title ProxyFactory
```

Comment:
I created this factory to provide `deploy...Proxy` methods allowing for [testing without hardhat upgrades and warning silencing](https://github.com/aragon/osx/blob/e7ba46026db96931d3e4a585e8f30c585906e1fc/packages/contracts/test/test-utils/proxy.ts#L1-L29).